pkg/agent/flowexporter/utils.go

```diff
@@ -27,3 +33,64 @@ func NewConnectionKey(conn *Connection) ConnectionKey {
 		strconv.FormatUint(uint64(conn.TupleOrig.Protocol), 10),
 	}
 }
+
+// ParseFlowCollectorAddr parses the flow collector address input for flow exporter and aggregator
+func ParseFlowCollectorAddr(addr string, defaultPort string, defaultProtocol string) (string, string, string, error) {
```

Comment:
I feel like this function should go under `pkg/util/flowexport` since it is common to the agent and the aggregator

Reply:
Thanks, addressed.